src/scripts/active-inactive-orgunit/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "active-inactive-orgunit",
+  "version": "1.0.0",
+  "description": "Sets each DataSet's period as true or false from a dataElementGroup",
```

Comment:
Maybe add something like 'depending on OU start and end date'. At the end this script is called active inactive org unit